templates/list.jsx

```diff
@@ -1,43 +1,56 @@
 import Adapt from 'core/js/adapt';
+import React from 'react';
 import { templates, classes, html, compile } from 'core/js/reactHelpers';
 
-export default function(model, view) {
-  const { _columns, _orderedList, _items } = model.toJSON();
+export default function List({ _columns, _orderedList, _items, ...props }) {
   const hasColumns = _columns > 1;
 
   return (
-    <div className='component__inner list__inner'>
-      {templates.component(model, view)}
-      <div className='component__widget list__widget'>
-        <div className={classes([
-          'list__container',
-          hasColumns && 'has-columns',
-          _orderedList && 'is-ordered-list'
-        ])} role='list'>
+    <div className="component__inner list__inner">
+      <templates.header {...props} />
+      <div className="component__widget list__widget">
+        <div
+          className={classes([
+            'list__container',
+            hasColumns && 'has-columns',
+            _orderedList && 'is-ordered-list'
+          ])}
+          role="list"
+        >
           {_items.map(({ _isActive, _imageSrc, alt, title, body }, index) =>
-            <div key={index} className={classes([
-              'list__item',
-              _isActive && 'is-animating'
-            ])} role='listitem' style={(hasColumns && Adapt.device.screenSize === 'large' && { width: `${100 / _columns}%` }) || null}>
-              <div className='list__item-inner'>
+            <div
+              key={index}
+              className={classes([
+                'list__item',
+                _isActive && 'is-animating'
+              ])}
+              role="listitem"
+              style={(hasColumns && Adapt.device.screenSize === 'large' && { width: `${100 / _columns}%` }) || null}
+            >
+              <div className="list__item-inner">
                 {!_imageSrc ?
-                  <div className='list__item-bullet'></div> :
-                  <div className='list__item-image-container'>
-                    <img className='list__item-image' src={_imageSrc} aria-label={alt} aria-hidden={!alt} />
+                  <div className="list__item-bullet"></div> :
+                  <div className="list__item-image-container">
+                    <img
+                      className="list__item-image"
+                      src={_imageSrc}
+                      aria-label={alt}
+                      aria-hidden={!alt}
+                    />
                   </div>
```

Comment:
https://github.com/adaptlearning/adapt_framework/blob/master/src/core/templates/image.jsx
```suggestion
                  <templates.image _src={_imageSrc} alt={alt} classNamePrefixes={[ 'list__item' ]} />
```